resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/VolcanoFeatureStep.scala

```diff
@@ -40,6 +41,10 @@ private[spark] class VolcanoFeatureStep extends KubernetesDriverCustomFeatureCon
   }
 
   override def getAdditionalPreKubernetesResources(): Seq[HasMetadata] = {
+    if (kubernetesConf.isInstanceOf[KubernetesExecutorConf]) {
+      logWarning("VolcanoFeatureStep#getAdditionalPreKubernetesResources() is not supported " +
+        "for executor.")
+    }
```

Comment:
According to the following PR description, there is no plan for `Volcano` to use `getAdditionalPreKubernetesResources` in the future?
```
Only spark.kubernetes.scheduler.volcano.podGroupTemplateFile is going to be supported in v3.3
```

Reply:
It only be called when building driver side feature steps.

I plused it. :)